clarify description (#1008)
mate-academy/py-game-models: checklist.md

)
```

> Normal example and works as well:

Reviewer: ```suggestion
Good example and it works well:

```

```python
Model.objects.create(
    field=(data["info"] if data["info"] else None)
)
```

> Bad example, don't use it:

Reviewer: ```suggestion
Bad example, avoid using it:

```

)
```

## 5. Use `.get()` method to check whether key defined in dictionary

Good example (`.get()` method returns `None` by default):
```python
guild = data.get("guild")
if guild:
    ...
```

Bad example:
```python
if "guild" in data:
    guild = data["guild"]
```

## 6. Improve your Code

### 1) Do not overload the context manager.

The context manager is needed to work with the file, in our case, to read data.

> Therefore, you need to have read file logic under the context manager block only (`json.load()` in our case).
> After that, you need to exit the block and continue working with received data.

Reviewer: ```suggestion
To read a file, you should use the logic within the context manager block only (e.g., `json.load()`). After finishing this block, you can continue working with the received data.
```
